box2d3-wasm/README.md

```diff
@@ -1,30 +1,30 @@
 # box2d3-wasm
 
-Let's try and use [Box2D v3][] on the web via [wasm][]!
+[Box2D v3][] ported to the web via [wasm][]!
 
-```bash
-git clone --recurse-submodules https://github.com/Birch-san/box2d3-wasm.git
-```
+## Online Demos
 
-## Install from npm
+Check out the [pyramid drop demo](https://box2d-v3-threading.netlify.app/?threading=1&pyramidHeight=64). You can change the size of the pyramid by editing the "pyramidHeight" URL parameter. Set it to 101 for a pyramid with 5,050 boxes!
 
-Install [Node.js and npm][].  
-Install [npm package](https://www.npmjs.com/package/box2d3-wasm) like so:
+Check out the [stress test demo](https://phaser-stress-test-box2d3-wasm.netlify.app/?threading=1). How many shapes can your computer simulate at 60fps? box2d3-wasm is the fastest Box2D implementation on the web.
 
-```bash
-npm i --save box2d3-wasm
-```
+## Web Usage 
```

Comment:
what was wrong with having subheadings? not every section here is orthogonal, we're allowed to have an information hierarchy

Reply:
I think when there's only two subsections, generally that's not enough, there should be at least three to justify doing sub-headings. That's just my opinion though. You could make "Usage" an h2 and then web usage and Node usage the h3s but they were just bold text before.

Reply:
I added the npm section as the third subsection of the "Usage" section.